src/main/java/constant/InputViewMessage.java

```diff
@@ -0,0 +1,31 @@
+package constant;
+
+public enum InputViewMessage {
+
+    ATTENDANCE_OPTION_PROMPT("오늘은 %s월 %s일 %s입니다. 기능을 선택해 주세요.%n"
+            + "1. 출석 확인%n"
+            + "2. 출석 수정%n"
+            + "3. 크루별 출석 기록 확인%n"
+            + "4. 제적 위험자 확인%n"
+            + "Q. 종료"),
```

Comment:
자바 15에 추가된 Text Blocks 을 사용해 보는 것은 어떨까요?
```suggestion
    ATTENDANCE_OPTION_PROMPT("""
        오늘은 %s월 %s일 %s입니다. 기능을 선택해 주세요.
        1. 출석 확인
        2. 출석 수정
        3. 크루별 출석 기록 확인
        4. 제적 위험자 확인
        Q. 종료"""
    ),
```

Reply:
와 정말 좋은 것 같아요!!!! 바로 적용했습니당 감사해요!